test/images/directx-device-plugin/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+windows/amd64/1809=mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
I think these need to be mcr.microsoft.com/windows:#### based on https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration 

Under requirements:
The container base image must be mcr.microsoft.com/windows:1809 or newer. Windows Server Core and Nano Server container images are not currently supported.

Reply:
The device plugin provides GPU access for other containers, but doesn't need GPU access itself. So nanoserver is fine.